From b172d0dc18f9fa3ff3c994be5d491eea84362d62 Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Fri, 20 Sep 2024 06:25:56 +0300
Subject: [PATCH] [FSW-9512] fs1: clear validation flag

This patch add support to clear validation pass
and set recovery flag when bootrom fail to load
both images and entering recovery.

Change-Id: I45ded4ed480f7c06f059260c62883505bd068e48
---

diff --git a/common/recovery.c b/common/recovery.c
index e548c44..c19b30f 100644
--- a/common/recovery.c
+++ b/common/recovery.c
@@ -53,6 +53,9 @@
 {
 	boot_rom_status_update(PROGRESS_BOOT_ROM_IN_RECOVERY);
 
+	/* Set recovery and clear validation pass flag */
+	boot_rom_validation_status(false);
+
 	boot_rom_set_recovery_status();
 
 	if (boot_rom_in_multi_die_mode())
